backend/src/api/base.clj

```diff
@@ -23,7 +23,8 @@
 
 (defn add-player
   "Adds a player to a game"
-  [game-id]
+  ([game-id] (add-player game-id {}))
```

Comment:
Ok,`add-player` is doing too many things already.

`create-game` should be the one creating a (playerless) game with its initial config, and then call add-player to simply to that: add a player. This would greatly simplify these functions (and I once again know its not specifically related to this issue, I might just open a separate issue now)